### ISSUE TITLE
Incorrect role is being assigned to combo box; it is impacting accessibility of the control. 

### ISSUE THREAD

**Issue by Kirank915** [brianreavis/selectize.js#2043]
**User Experience:**
Visually impaired people who use screen reader will not be able to know the purpose of control if the incorrect role is defined as textbox for control.

Note: 
User credentials should NOT be included in the bug.

**Repro Steps:**
1. Open URL:[ aka.ms/ipamui ](http://aka.ms/ipamui)and login with valid credentials.
2. Home page appears
3. Navigate to search field on top header and search Ip address "100.80.0.0/15" and navigate to "Request details" tab.
4. Navigate all controls under the "Request details" page and navigate to table and select "Ip Address link".
5. Navigate all controls on main page and navigate to header controls and select "IPV6" and navigate all controls under "IPV6" and select "Edit" button on table.
6. Navigate to all controls under "Edit allocation" dialog like "ID and prefix."            
7. Verify the issue.

**Actual Result:**
Incorrect role as textbox is defined for the "Reuse scope, tag name, tag value" controls present on the "edit allocation dialog. Screen reader is announcing the information as "Reuse scope none dash edit".

**Expected Result:**
Correct role as "combo box" should be defined for the "Reuse scope, tag name, tag value" controls present on the "edit allocation dialog. Screen reader should announce the information as "Reuse scope combo box collapsed/expanded".

**Comment by Kirank915** [comment]
Please find below comments from the IP Address Management Engineering product team:

We use a 3rd-party JS library 'JQuery.Selectize' to render all <select> boxes to combo boxes. In our code, we just need to declare a <select> HTML element in the page and call selectize() method in the JS. The library will do all the magic things for us. See:

<img width="407" alt="image" src="https://github.com/selectize/selectize.js/assets/117266902/1b5c3d79-e7a8-41bd-b6e5-8c59878d309e">

So, coming to the IPAM web site, the original select box will be covered by the combo box that generated by the library. For instance
Although I have set the select's role to "combobox", because the library-generated elements will cover it. The screen reader cannot hit it.
Meanwhile, we cannot assign the role to the JQuery.Selectize either as it doesn't provide this API.

<img width="748" alt="image" src="https://github.com/selectize/selectize.js/assets/117266902/9d4626b7-6e27-453b-b003-fd8fa53f21a4">
